lib/mix/tasks/compile.make.ex

```diff
@@ -83,6 +83,11 @@ defmodule Mix.Tasks.Compile.ElixirMake do
   and `mix test` commands.
   """
 
+  @exec_notfound_msg """
```

Comment:
I think other messages are in module attributes for easier selection, but probably this one doesn't require to be here. :bowtie: